worldedit-core/src/main/java/com/sk89q/worldedit/util/paste/EngineHubPaste.java

```diff
@@ -49,47 +51,49 @@ private PasteTask(String content, PasteMetadata metadata) {
 
         @Override
         public URL call() throws IOException, InterruptedException {
-            URL initialUrl = HttpRequest.url("https://paste.enginehub.org/signed_paste");
-
-            HttpRequest requestBuilder = HttpRequest.get(initialUrl);
-
-            requestBuilder.header("x-paste-meta-from", "EngineHub");
-            if (metadata.name != null) {
-                requestBuilder.header("x-paste-meta-name", metadata.name);
-            }
-            if (metadata.author != null) {
-                requestBuilder.header("x-paste-meta-author", metadata.author);
+            try (HttpClient client = HttpClient.newHttpClient()) {
+                HttpRequest.Builder signRequestBuilder = HttpRequest.newBuilder()
+                        .uri(URI.create("https://paste.enginehub.org/signed_paste_v2"))
+                        .header("x-paste-meta-from", "EngineHub");
+
+                if (metadata.name != null) {
+                    signRequestBuilder = signRequestBuilder.header("x-paste-meta-name", metadata.name);
```

Comment:
Unnecessary reassignment. Builder can be mutated in place.

Reply:
Fixed.